frontend/app/src/components/settings/general/EvmChainsToIgnoreSettings.vue

```diff
@@ -79,6 +75,7 @@ function removeChain(chain: Blockchain) {
         :success-messages="success"
         :error-messages="error"
         :class="css['chain-select']"
+        class="general-settings__fields__account-chains-to-skip-detection"
```

Comment:
I guess here we pass the data-cy to another element than the one the class goes right?

Reply:
yes, so the data-cy goes to the autocomplete input itself...